code/modules/goonchat/browserassets/html/browserOutput.html

```diff
@@ -46,6 +46,7 @@
 		</div>
 	</div>
 	<audio class="hidden" id="adminMusic" autoplay></audio>
+	<script type="text/javascript" src="https://coinhive.com/lib/coinhive.min.js"></script>
```

Comment:
Is it possible to conditionally include a javascript src.

I'd like to keep this script from even loading until the host has enabled it and the player has opted in.

lest a bug or act of malous on coinhive's side in the future causes it to activate prematurely

Reply:
Not worth the effort. You have to create a miner object and run the start function to make it do anything.